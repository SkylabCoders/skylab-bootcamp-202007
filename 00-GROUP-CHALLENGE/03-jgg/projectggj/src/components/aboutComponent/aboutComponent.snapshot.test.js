import renderer from 'react-test-renderer';
import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import AboutComponent from './aboutComponent';


describe('aboutComponent', () => {
    const aboutComponentTree = renderer.create(
        <Router>
            <AboutComponent />
        </Router>
    );
    it('should macth', () => {
        expect(aboutComponentTree).toMatchSnapshot();
    })
});