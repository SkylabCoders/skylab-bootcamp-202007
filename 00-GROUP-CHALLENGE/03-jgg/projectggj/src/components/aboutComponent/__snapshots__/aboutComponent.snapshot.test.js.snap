// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`aboutComponent should macth 1`] = `
Array [
  <h1
    className="titleAbout"
  >
    Meet the development team
  </h1>,
  <section
    className="mainContainerAbout"
  >
    <div
      className="usAbout"
    >
      <img
        alt="profile jordi"
        className="profileImage"
        src="https://media-exp1.licdn.com/dms/image/C5603AQEF9iyRfUz82Q/profile-displayphoto-shrink_200_200/0?e=1602115200&v=beta&t=04dUdp6lbgARNyQT4sIUDRjan8pk16SVwj9A1YDoFe8"
      />
      <h3>
        Jordi Mu√±oz
      </h3>
      <p>
        Front-End y Back-End Developer
      </p>
      <div
        className="containerAbout"
      >
        <a
          href="https://www.linkedin.com/in/jordi-mu%C3%B1oz-mallofr%C3%A9-2b1043150/"
        >
          <img
            alt="linkedin logo"
            src="https://icons-for-free.com/iconfiles/png/512/linked+linkedin+logo+social+icon-1320191784782940875.png"
          />
        </a>
        <a
          href="https://github.com/gemmas95?tab=repositories"
        >
          <img
            alt="github logo"
            src="https://image.flaticon.com/icons/png/512/25/25231.png"
          />
        </a>
      </div>
    </div>
    <div
      className="usAbout"
    >
      <img
        alt="profile gabriel"
        className="profileImage"
        src="https://media-exp1.licdn.com/dms/image/C5103AQFFJO_4j-gK4A/profile-displayphoto-shrink_200_200/0?e=1602115200&v=beta&t=9Hknm4pleqf4Hd0I0UjvqR8n_Ujy7xGWi4f9D8Vi9MY"
      />
      <h3>
        Gabriel Penalva
      </h3>
      <p>
        Web Developer
      </p>
      <div
        className="containerAbout"
      >
        <a
          href="https://www.linkedin.com/in/gabriel-penalva-iba%C3%B1ez-874542120/"
        >
          <img
            alt="linkedin logo"
            src="https://icons-for-free.com/iconfiles/png/512/linked+linkedin+logo+social+icon-1320191784782940875.png"
          />
        </a>
        <a
          href="https://github.com/gemmas95?tab=repositories"
        >
          <img
            alt="github logo"
            src="https://image.flaticon.com/icons/png/512/25/25231.png"
          />
        </a>
      </div>
    </div>
    <div
      className="usAbout"
    >
      <img
        alt="profile gemma"
        className="profileImage"
        src="https://media-exp1.licdn.com/dms/image/C5603AQFh0WJ5yijZ9g/profile-displayphoto-shrink_200_200/0?e=1602115200&v=beta&t=M6gWcjNJIezUHM3teUTQjH2QDuf5A2RxQcDvOm71m8Q"
      />
      <h3>
        Gemma Sanz
      </h3>
      <p>
        FullStack Web Developer
      </p>
      <div
        className="containerAbout"
      >
        <a
          href="https://www.linkedin.com/in/gemma-sanz-rabad%C3%A1n-44367a13b/"
        >
          <img
            alt="linkedin logo"
            src="https://icons-for-free.com/iconfiles/png/512/linked+linkedin+logo+social+icon-1320191784782940875.png"
          />
        </a>
        <a
          href="https://github.com/gemmas95?tab=repositories"
        >
          <img
            alt="github logo"
            src="https://image.flaticon.com/icons/png/512/25/25231.png"
          />
        </a>
      </div>
    </div>
  </section>,
]
`;
